images/centos7/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM registry.centos.org/centos:7
+
+RUN yum -y install http://yum.theforeman.org/client/$foreman_version/el7/x86_64/foreman-client-release.rpm
```

Comment:
doesn't `foreman_version` need to be defined as an `ARG`?

Reply:
It does not for this work. I can add it if we prefer.

Reply:
If it works, it works.